html/janus.js

```diff
@@ -108,9 +108,9 @@ Janus.useDefaultDependencies = function (deps) {
 				fetchOptions.headers['Content-Type'] = 'application/json';
 			}
 			if(options.withCredentials !== undefined) {
-				fetchOptions.credentials = options.withCredentials === true ? 'include' : (options.withCredentials ? options.withCredentials : 'omit');
```

Comment:
I'm not sure changes like this are ok. The API requires that value to be `true`, but that check will succeed for anything that in a check resolves to `true`, so basically anything that is not `0`, `null` or `undefined` I guess? I'm ok with simplifying the code, but not at the expense of a weaker API.